apis/Google.Cloud.Diagnostics.AspNet/Google.Cloud.Diagnostics.AspNet.sln

```diff
@@ -3,15 +3,15 @@ Microsoft Visual Studio Solution File, Format Version 12.00
 # Visual Studio 14
 VisualStudioVersion = 14.0.25420.1
 MinimumVisualStudioVersion = 10.0.40219.1
-Project("{8BB2217D-0F2D-49D1-97BC-3654ED321F3B}") = "Google.Devtools.AspNet", "Google.Devtools.AspNet\Google.Devtools.AspNet.xproj", "{D85333C3-F612-4D92-8ECD-E674A73AEC8F}"
+Project("{8BB2217D-0F2D-49D1-97BC-3654ED321F3B}") = "Google.Cloud.Diagnostics.AspNet", "Google.Cloud.Diagnostics.AspNet\Google.Cloud.Diagnostics.AspNet.xproj", "{D85333C3-F612-4D92-8ECD-E674A73AEC8F}"
 EndProject
-Project("{8BB2217D-0F2D-49D1-97BC-3654ED321F3B}") = "Google.Devtools.AspNet.Tests", "Google.Devtools.AspNet.Tests\Google.Devtools.AspNet.Tests.xproj", "{575CD5C0-C69D-495E-90FD-028BE94178E4}"
+Project("{8BB2217D-0F2D-49D1-97BC-3654ED321F3B}") = "Google.Cloud.Diagnostics.AspNet.Tests", "Google.Cloud.Diagnostics.AspNet.Tests\Google.Cloud.Diagnostics.AspNet.Tests.xproj", "{575CD5C0-C69D-495E-90FD-028BE94178E4}"
 EndProject
 Project("{8BB2217D-0F2D-49D1-97BC-3654ED321F3B}") = "Google.Devtools.Clouderrorreporting.V1Beta1", "..\Google.Devtools.Clouderrorreporting.V1Beta1\Google.Devtools.Clouderrorreporting.V1Beta1\Google.Devtools.Clouderrorreporting.V1Beta1.xproj", "{4AB959C8-953B-41BA-9C8F-392002F6A4F6}"
```

Comment:
Remove this project entirely from the solution? I'm not sure what it's doing here.